### DISCUSSION TITLE
Stream from Smartphone (Android) to Chromecast

### DISCUSSION THREAD

**Discussion by streiman** [mediacms-io/mediacms#421] [Q&A]
I just installed Mediacms on my homeserver for testing - and I am really impressed. But I have a problem / question. When I watch a video on my smartphone (Android, Pixel 6, Chrome Browser) I get the button to stream the video to my chromecast on my TV. But when I select it, the TV only shows the "Chrome" logo, no video is playing.

Is this a normal behaviour or is there anything I can do to enable casting to my TV?

If I stream from Chrome on my notebook everything works fine.

**Comment by mgogoulos** [comment]
Hi @streiman ,
I'm not sure it is possible to stream to chromecast, maybe it's a default option of the video.js player MediaCMS uses, but it's not something that has been tested. I'll now convert this to an issue (slightly changing the title too)
